ingress/controllers/nginx/controller.go

```diff
@@ -580,6 +581,12 @@ func (lbc *loadBalancerController) getUpstreamServers(data []interface{}) ([]*ng
 				continue
 			}
 
+			rl, err := ratelimit.ParseAnnotations(ing)
+			glog.V(3).Infof("nginx rate limit %v", rl)
+			if err != nil {
+				glog.V(3).Infof("error reading rate limit annotation in Ingress %v/%v: %v", ing.GetNamespace(), ing.GetName(), err)
```

Comment:
things like this feel event worthy, if it's easy enough to send an event. If the user thought they specified the right annotations but didn't, they should know their backend isn't getting ratelimited without having to debug a bunch and look around in the logs